### DISCUSSION TITLE
How can i change my weather location on waybar?

### DISCUSSION THREAD

**Discussion by notsim1** [JaKooLit/Hyprland-Dots#473] [Q&A] [answered]
My weather temperature in the widget is incorrect and when i idle my mouse on the icon, it shows: **not found: (incorrect weather)**. How can i change it?

**Comment by JaKooLit** [comment, marked as answer]
https://github.com/JaKooLit/Hyprland-Dots/wiki/TIPS#%EF%B8%8F-weather-app-related-for-waybar-and-hyprlock
